courses/machine_learning/deepdive2/supplemental/solutions/question_answering_rnn.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>Use the <a href="https://www.python.org/dev/peps/pep-0008/#imports" target="_blank">pep8</a> convention to format the imports</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>